db/blob/blob_file_reader.h

```diff
@@ -8,9 +8,11 @@
 #include <cinttypes>
 #include <memory>
 
+#include "db/blob/blob_read_request.h"
 #include "file/random_access_file_reader.h"
 #include "rocksdb/compression_type.h"
 #include "rocksdb/rocksdb_namespace.h"
+#include "table/multiget_context.h"
```

Comment:
Probably unnecessary now `BlobReadRequest` got moved to its own header

Reply:
Done